### PR TITLE
Updated students.yml .

### PR DESCRIPTION
**Pull request by doluz**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Added details in student.yml 
- GCI Task Instance : https://codein.withgoogle.com/dashboard/task-instances/6252390464356352/

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -11,6 +11,14 @@
   lat: 22.484401
   lng: 88.314114
 
+- name: Sidhdharth
+  github: sidhdharths
+  image: sidhdharth.jpg80.2707184
+  twitter: sidhdharth
```
Comment (abishekvashok) [inline _data/students.yml]: Your twitter username seems incorrect pls double check.

File: _data/students.yml [hunk 2]
```diff
@@ -11,6 +11,14 @@
   lat: 22.484401
   lng: 88.314114
 
+- name: Sidhdharth
+  github: sidhdharths
+  image: sidhdharth.jpg80.2707184
```
Comment (abishekvashok) [inline _data/students.yml]: This image was not uploaded...

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please add your images and fix as stated:

**Comment by robbyoconnor** [comment]
You don't have the image here @sidhdharths...

File: _data/students.yml [hunk 3]
```diff
@@ -11,6 +11,14 @@
   lat: 22.484401
   lng: 88.314114
 
+- name: Sidhdharth
+  github: sidhdharths
+  image: sidhdharth.jpg
```
Comment (robbyoconnor) [inline _data/students.yml]: Did you forget to upload this?

**Comment by robbyoconnor** [comment]
@sidhdharths why did you close this?

**Comment by abishekvashok** [comment]
@robbyoconnor nice work

**Pull request merged**